docs/developer-docs/developer-welcome.md

```diff
@@ -0,0 +1,181 @@
+
+The biggest new privilege is that you can now link new hashes to games by loading the game in an RA emulator and linking it.
+
+If creating a new page, select "New Title" and write an appropriate name in the text box. Check if there are any junk pages labeled ~Z~ which can be recycled. 
+
+::: info
+Further reading:
+
+[Creating Entries](/guidelines/developers/creating-entries)
+
+[Achievements for ROM Hacks](/guidelines/content/achievements-for-rom-hacks)
+
+[Subsets](/guidelines/content/subsets)
+
+[Working with the Right ROM](/guidelines/content/working-with-the-right-rom)
+:::
+
+![Creating a game page with an unidentified hash](/public/game-page-creation.png)
+
+### Code Notes
+
+You are now able to overwrite and delete any existing code notes. Be careful not to overwrite other’s notes without reason, if you are making improvements or correcting a note that is an acceptable reason.
+
+### Achievements
+
+Developers can directly make edits to achievements in Core.
+
+You can now freely promote and demote any achievements from Unofficial and Core through the emulator but it is much easier to do this through the site.
+
+## Site Updates
+
+You now have access to many more tools on the site to help you as a developer.
+
+### Want to Develop Games
+
+Developers have the ability to add games to a Want to Develop list. Sets can be added through the Dev dropdown box on the game page.
+
+![Want to Develop Games button](/public/want-to-develop.png)
+
+### Game Page
+
+#### Creating Forum Topics
+
+When you go to claim a set, you no longer need to create a forum topic separately. The Make Claim button will automatically create a forum topic for you if one does not exist. 
+
+Keep in mind that you should NOT create forum topics without a valid reason, such as claiming a set. Otherwise, you will be the only person (besides admins) who can edit the top post, which is usually where important information such as linked hashes or set notes is placed.
+
+#### Dev Menu
+
+You now have full access to the Dev dropdown menu.
+
+- **Manage Core/Unofficial Achievements**: Promotion/demotion, typing, and ordering of achievements. 
+
+- **Manage Leaderboards**: Resetting entries, ordering, and deletion of leaderboards
+    - Please read [Editing Leaderboards](/guidelines/content/editing-leaderboards) before making edits or resetting entries for leaderboards that are not your own.
+
+- **Manage Hashes**: Editing hash labels, Editing RAPatches URLs, and unlinking hashes
+
+- **Subscribe to Tickets**: Useful if you decide to work on inactive developer sets and want to track any future tickets
+
+- **Want to Develop**: Adds/Removes the game on your Want to Develop list
+
+![Dev menu](/public/dev-menu.png)
+
+#### Adding Hubs and Related Games
+
+Hubs and related games can be added to game pages by entering the page IDs and clicking “Add”. Hubs managed by the QA and Art teams should not be added/removed, send a message to the team instead.

```

Comment:
Add DevComp to the teams in this list since we have our own hubs as well that shouldn't be tampered with